scanapi/config_loader.py

```diff
@@ -28,7 +33,16 @@ def __init__(self, stream: IO) -> None:
 
 
 def construct_include(loader: Loader, node: yaml.Node) -> Any:
-    """Include file referenced at node."""
+    """Include file referenced at node.
+
+        Args:
+            loader [class.Loader]: fill me!
```

Comment:
@Pradhvan are you going to leave the `fill me` for now and fill it later? Or are you going to fill it still in this PR? I was waiting for it to approve the PR. But we can leave it for later, if you prefer :)

Reply:
Hey! @camilamaia will finish this PR and update the `fill me` this weekend. Then we can merge this. 😄